test/controllers/offer.controller.test.js

```diff
@@ -946,6 +946,8 @@ describe('Offer Controller', () => {
               expect(res.body.success).to.be.eql(true);
               expect(res.body).to.have.property('offers');
               expect(res.body.offers[0]).to.have.property('concern');
+              expect(res.body.offers[0].concern.length).to.be.eql(0);
+              expect(res.body.offers[1]).to.have.property('concern');
```

Comment:
Not sure why you used offers[1] instead of 0 here. I believe that if we test for the index of 0 only, it will cover for 1 too. 

```suggestion
              expect(res.body.offers[0]).to.have.property('concern');
```

Reply:
OR this will work too
```suggestion
              expect(res.body.offers[1].concern.length).to.be.eql(0);
```

Reply:
exists already in line `948`